src/misc/write-docs/index.md

```diff
@@ -416,8 +416,8 @@ With a clear picture of how the structure of content will change in a given rele
 * **General checks**
     * Ensure source and destination URLs are separated by a single tab, not spaces
     * Ensure all target URLs end with a trailing `/` for example:
-        * https://merginmaps.com/docs/howto/input_ui (bad)
-        * https://merginmaps.com/docs/howto/input_ui/ (good)
+        * `https://merginmaps.com/docs/howto/input_ui` (bad)
+        * `https://merginmaps.com/docs/howto/input_ui` (good)
```

Comment:
Hi @PeterPetrik ,

That line was meant to have a trailing slash - that was the main idea to convey here :)

Reply:
sorry, let me fix it